wlutil/br/br.py

```diff
@@ -121,22 +121,22 @@ def generateBootScriptOverlay(script, args):
         # The buildroot repo has a pre-built overlay with a custom S99run
         # script that init will run last. This script will run the /firesim.sh
         # script at boot. We just overwrite this script.
-        scriptDst = os.path.join(overlay, 'firesim.sh')
+        scriptDst = overlay / 'firesim.sh'
         if script != None:
-            wlutil.run(['cp', str(script), str(scriptDst)])
+            shutil.copy(script, scriptDst)
         else:
-            wlutil.run(['rm', scriptDst])
+            scriptDst.unlink()
             # Create a blank init script because overlays won't let us delete stuff
             # Alternatively: we could consider replacing the default.target
             # symlink to disable the firesim target entirely
-            wlutil.run(['touch', scriptDst])
+            scriptDst.touch()
         
-        wlutil.run(['chmod', '+x', scriptDst])
+        scriptDst.chmod(0o777)
```

Comment:
Is there a reason why the script has to be world-writable instead of `0755`?

Reply:
because unix permissions confuse me :). I switched to 755.